packages/amplify_core/lib/src/types/auth/sign_in/auth_provider.dart

```diff
@@ -22,7 +22,8 @@ class AuthProvider
   /// configuring the provider, while [issuer] must match the `iss` value of
   /// the provider's ID token.
   const AuthProvider.oidc(this.name, String issuer)
-      : _identityPoolProvider = issuer;
+      : _identityPoolProvider = issuer,
+        displayName = null;
```

Comment:
Should allow taking a displayName too. `name` is just however it was registered in Cognito, but it may not be display-worthy.